ab_tool/canvas.py

```diff
@@ -90,6 +92,19 @@ def _get_installed_intervention_point_urls(self):
         return installed_intervention_point_urls
 
 
+def get_unsorted_students(request, experiment):
+    request_context = get_canvas_request_context(request)
+    course_id = get_lti_param(request, "custom_canvas_course_id")
+    try:
+        enrollments = list_users_in_course_users(
+                request_context, course_id, None, enrollment_type="student").json()
+    except RequestException as exception:
+        handle_canvas_error(exception)
+    existing_student_ids = set(s.id for s in experiment.students.all())
+    return [{"student_id": i["sis_user_id"], "lis_person_sourcedid": i["sis_user_id"]}
```

Comment:
The intention for the return values here was to match the keys in the ExperimentStudent model.  This output is simplified in a later pull